azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeLocalOptions.java

```diff
@@ -234,4 +237,24 @@ public CallCompositeLocalOptions setCaptionsOptions(
         this.captionsOptions = captionsOptions;
         return this;
     }
+
+    /* <END_CALL_FOR_ALL>
+    \**
+     * Get the boolean value for on call end terminate for all.
+     * @return The boolean that is currently set.
+     *\
+    public boolean isOnCallEndTerminateForAll() {
+        return this.onCallEndTerminateForAll;
+    }
+
+    \**
+     * Set a boolean to be used.
```

Comment:
Does not sound good to me.
How about:
'Configure if call should end for all participants when current user hangs up.'